lib/engine/round/operating.rb

```diff
@@ -657,16 +657,21 @@ def min_token_price(token)
       end
 
       def token_price_ability(token, hex)
-        return [0, :teleport] if @teleported
+        if @teleported
```

Comment:
this is now a dangerous method 

rename to

def adjust_token_price_ability! 

maybe?